content/en/service_management/workflows/private_actions/_index.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Private Actions
+kind: Documentation
+disable_toc: false
+further_reading:
+- link: "service_management/app_builder/connections"
+  tag: "Documentation"
+  text: "App Builder Connections"
+- link: "service_management/workflows/connections"
+  tag: "Documentation"
+  text: "Workflow Connections"
+- link: "service_management/workflows/private_actions/private_action_credentials"
+  tag: "Documentation"
+  text: "Handling Private Action Credentials for Workflow Automation"
+- link: "service_management/app_builder/private_actions/private_action_credentials"
+  tag: "Documentation"
+  text: "Handling Private Action Credentials for App Builder"
+---
+
+{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSc_AEtxd8MPHUFyYIbX3hWEVF18iSWuk7kmA0PtlOPgo0pi3w/viewform" btn_hidden="false" header="Join the Beta!">}}
+Private Actions are in beta. Use this form to request access today.
+{{< /callout >}}
+
+## Overview
+
+Private actions allow your Datadog workflows and apps to interact with services hosted on your private network without exposing your services to the public internet. To use private actions, you must use Docker to install a private action runner on a host in your network and pair the runner with a Datadog Connection.
```

Comment:
```suggestion
Private actions allow your Datadog workflows and apps to interact with services hosted on your private network without exposing your services to the public internet. To use private actions, you must use Docker to install a private action runner on a host in your network, then pair the runner with a Datadog Connection.
```

just a suggestion because the sentence is long enough to be a little confusing